configure.ac

```diff
@@ -784,6 +802,9 @@ echo "Event handlers:"
 AM_COND_IF([ENABLE_SAMPLEEVH],
 	[echo "    Sample event handler:  yes"],
 	[echo "    Sample event handler:  no"])
+AM_COND_IF([ENABLE_RABBITMQEVH],
+  [echo "    RabbitMQ event handler:yes"],
+  [echo "    RabbitMQ event handler:no"])
```

Comment:
The indentation for these two echos seems wrong, maybe it's spaces instead of a tab?